api/turing/service/router_deployment_service_test.go

```diff
@@ -271,9 +259,6 @@ func TestDeployEndpoint(t *testing.T) {
 		BaseService: &cluster.BaseService{
 			Name:      fmt.Sprintf("%s-enricher-%d", routerVersion.Router.Name, routerVersion.Version),
 			Namespace: testNamespace,
-			Labels: map[string]string{
-				"env": envType,
-			},
```

Comment:
Removed these from the expected responses since we were just 'expecting' a mocked response though in reality the actual method doesn't even actually set the `envType` as the `env` label.